transcom/mymove
# INT-B-21551 NTS-R now has estimated prices on service items

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1022729&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Calculates estimated prices on NTS-R service items.

### How to test

1. Create an NTS-R move.
2. Select Vender: GHC Prime contractor and submit the move to prime.
3. Navigate to MTO page and verify estimated prices are there.
